packages/strapi-plugin-content-manager/admin/src/containers/EditView/Header.js

```diff
@@ -28,23 +28,28 @@ const Header = () => {
     resetData,
     setIsSubmitting,
     slug,
+    clearData,
   } = useDataManager();
+  const isSingleType = useRouteMatch('/plugins/content-manager/singleType');
 
   const currentContentTypeMainField = get(
     layout,
     ['settings', 'mainField'],
     'id'
   );
+  const currentContentTypeName = get(layout, ['schema', 'info', 'name']);
+  const apiId = layout.uid.split('.')[1];
```

Comment:
Is this temporary?

Reply:
Yes, it was. However, the API is updated so I can use the new API response to use the real API ID.

Reply:
Works for me then!